resources/warnings.json

```diff
@@ -3858,7 +3858,8 @@
     "url": "https://jenkins.io/security/advisory/2019-04-03/#SECURITY-1042",
     "versions": [
       {
-        "pattern": ".*"
+        "lastVersion": "1.3",
+        "pattern": "jenkins-cloudformation-plugin-[0-1][.][0-9]+"
```

Comment:
```suggestion
        "lastVersion": "1.2",
        "pattern": "(0|1[.][02])(|[.-].+)"
```

These patterns are for version numbers.

Additionally, version 1.3 was never released: https://repo.jenkins-ci.org/artifactory/releases/org/jenkins-ci/plugins/jenkins-cloudformation-plugin/